projects/experimental/components/file-uploader/file/file.component.html

```diff
@@ -0,0 +1,72 @@
+<div
+    *ngIf="mode === 'cell'"
+    class="t-cell"
+>
+    <tui-swipe-actions>
```

Comment:
This would make it dependent on addon-mobile, we want to avoid that

Reply:
fixed